src/components/SearchBy/SearchBy.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './searchby.scss';
+
+const cx = classNames.bind(styles);
+
+const SearchBy = () => (
+  <div className={styles.searchBy}>
+    <p className={styles.title}>Search by</p>
+    <ul className={styles.list}>
+      <li className={cx(styles.item, styles.active)}>
```

Comment:
If you use classNames.bind you can pass only names of classes: cx('item', 'active')